IdIOHandlerWebSocketSSL.pas

```diff
@@ -150,7 +150,10 @@ implementation
 {$IF DEFINED(MSWINDOWS)}
   Winapi.Windows,
 {$ELSE}
-  IdSSLOpenSSLHeaders, FMX.Platform,
+  IdSSLOpenSSLHeaders,
+{$IFDEF DEFINED(ANDROID) or DEFINED(MACOS)}
```

Comment:
Same issue here.
{$IFDEF DEFINED... is incorrect}

It should be {$IF DEFINED...}